homeassistant/components/tuya/light.py

```diff
@@ -109,7 +120,10 @@ def turn_on(self, **kwargs):
         ):
             self._tuya.turn_on()
         if ATTR_BRIGHTNESS in kwargs:
-            self._tuya.set_brightness(kwargs[ATTR_BRIGHTNESS])
+            scaled_brightness = _scale(
+                kwargs[ATTR_BRIGHTNESS], NORMAL_LIGHT_RANGE, TUYA_LIGHT_TURN_ON_RANGE
+            )
+            self._tuya.set_brightness(round(scaled_brightness))
```

Comment:
And this should be: `self._tuya.set_brightness(round(scaled_brightness, 1))`

Reply:
```suggestion
            self._tuya.set_brightness(round(scaled_brightness, 1))
```